File: Raven.Abstractions/Data/FacetResults.cs
using System;
using System.Collections.Generic;

namespace Raven.Abstractions.Data
{
	public class FacetResults
	{
		/// <summary>
		/// A list of results for the facet.  One entry for each term/range as specified in the facet setup document.
		/// </summary>
		public Dictionary<string, FacetResult> Results { get; set; }

		/// <summary>
		/// Indicates how much time it took to process facets on server.
		/// </summary>
        public TimeSpan Duration { get; set; }

		public FacetResults()
		{
			Results = new Dictionary<string, FacetResult>();
		}
	}
}

File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/data/FacetResults.java
package net.ravendb.abstractions.data;

import java.util.HashMap;
import java.util.Map;

public class FacetResults {
  private Map<String, FacetResult> results;

  /**
   * A list of results for the facet.  One entry for each term/range as specified in the facet setup document.
   * @return
   */
  public Map<String, FacetResult> getResults() {
    return results;
  }

  public void setResults(Map<String, FacetResult> results) {
    this.results = results;
  }

  public FacetResults() {
    results = new HashMap<>();
  }
}
